localization/english/bookmark/bookmark_605_l_english.yml

```
@@ -1,11 +1,34 @@
 ﻿l_english:
  bm_the_frozen_throne:0 "The Frozen Throne"
- bm_the_frozen_throne_desc:0 "PH"
+ bm_the_frozen_throne_desc:0 "The apocalyptic Third War has come to an end and with it the Burning Legion's reign of chaos. The demon-lord Archimonde has been destroyed and the demonic invasion repelled, Azeroth was saved. Now, in the aftermath of the war, the night elves are rebuilding their wounded civilisation in Kalimdor, not far from their new neighbors exiled from the Eastern Kingdoms. Both humans and orcs settled in the continent to found their respective realms of Theramore and Durotar. To the east, Lordaeron is now dominated by the undead and ruled over by its new king, Arthas, in the name of the Lich King.
+ 
+ The world is not yet in peace, however, for the Lich King is weakened and the Legion does not wish his treason to stay unpinished. Remaining dreadlords fight to keep control of the Scourge, while the fallen night-elf Illidan Stormrage has been recruited by Kil'Jaeden to destroy the Frozen Throne itself. The fate of the Scourge is set and the Frozen Throne is the prize, a prize that may change the face of Azeroth forever.
+ 
+ Will the Lich King finally free himself from his demonic masters to pursue his own evil agenda ? What fate awaits his champion, prince Arthas ? What does Illidan Stormrage plans to vainquish this terrible foe ?  "
```

- still needs to be corrected
- still needs to be added at the end of line/paragraph 3
```suggestion
\n\nWill the Lich King finally free himself from his demonic masters to pursue his own evil agenda? What fate awaits his champion, prince Arthas? What does Illidan Stormrage plan to vanquish this terrible foe?"
```